runtime/queries/_gjs/highlights.scm

```diff
@@ -0,0 +1,5 @@
+[
+  (glimmer_opening_tag)
+  (glimmer_closing_tag)
+] @constant.builtin
```

Comment:
Are these similar to JSX opening and closing brackets? Those are highlighted as `@punctuation.bracket`

Reply:
![image](https://github.com/helix-editor/helix/assets/110528300/0068caa8-48cb-48d0-9145-9f90a88064f5)

its just the entire thing

Reply:
Ah ok, then `@tag` might be a better capture since it matches HTML tags

Reply:
how about `@constructor`?